app/main.py

```diff
@@ -1,8 +1,34 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self: list, cars: list) -> list:
+        return [car for car in cars if car.clean_mark < self.clean_power]
```

Comment:
The `serve_cars` method should calculate and return the income from washing the cars, not just filter them. You need to implement the logic to calculate the washing price for each car that is washed and sum these prices to get the total income. Refer to the task description for the formula to calculate the washing price.